contracts/DePayRouterV1OneInchSwap01.sol

```diff
@@ -86,7 +87,8 @@ contract DePayRouterV1OneInchSwap01 {
                 amounts[2]
             )
         );
+        return true;
     }
-    return false;
+    revert("DePayRouterV1OneInchSwap01: Unable to process swap");
```

Comment:
This is dead code. The interpreter will never arrive here as either `path[0] == ETH` or it's not (See: https://github.com/DePayFi/depay-ethereum-router/pull/27/files#diff-1c232ae7cc0dc1e2ad1888cdb53cad974d2613884211900772080e166a514017R65)

ending up either in the first half of the `if`-case or in the `else` part. As both return individually, this part of the smart contract will never be reached.

Reply:
Then, as both paths of the if-case lead to `return true` it can be dried to just returning true all together at the end of the method call, see: Uniswap v2 plugin: https://github.com/DePayFi/depay-ethereum-router/blob/master/contracts/DePayRouterV1Uniswap01.sol#L96

Reply:
Also, OneSplit.swap raises/reverts individually when swap failed, so we actually do not need to cover this in the plugin.

See: https://github.com/1inch/1inchProtocol/blob/master/contracts/OneSplitAudit.sol#L287-L290
https://github.com/1inch/1inchProtocol/blob/master/contracts/OneSplitAudit.sol#L320
https://github.com/1inch/1inchProtocol/blob/a7781cf9aa1cc2aaa5ccab0d54ecbae1327ca08f/contracts/UniversalERC20.sol#L24

Reply:
I think, `return true` would be ignored since #25 . I will add code to cover error states of 1inch swap.